BuildResidentialHPXML/measure.rb

```diff
@@ -5036,6 +5041,11 @@ def self.set_hvac_control(hpxml, args, epw_file, weather)
 
     end
 
+    # Natural Ventilation
```

Comment:
I'm a little unsure where this input should live, so I have tentatively added it to HVACControl

@shorowit thoughts?

Reply:
My initial thought is to add it next to the `NaturalVentilationAvailabilityDaysperWeek` element; see https://openstudio-hpxml.readthedocs.io/en/latest/workflow_inputs.html#hpxml-windows for where that description.

Reply:
Yep I think that would make sense, I'll make the change